src/ts/client/grapesjs/keymaps.ts

```diff
@@ -32,6 +37,43 @@ function toggleCommand (editor: Editor, name: string): void {
   }
 }
 
+/**
+ * Opens the Traits Manager panel.
+ * @param editor The editor.
+ */
+function openTraitsManager(editor: Editor): void {
+  editor.Panels.getButton('views', 'open-tm').set('active', true)
```

Comment:
In grapesjs code I found this, doesn't it mean that there is a command for that?
https://github.com/GrapesJS/grapesjs/blob/b247a580751b626017c44ddac2981b9dff6772df/src/commands/index.ts#L60

I'd rather avoid programatically click on the button

Reply:
This is indeed the command to open the Traits Manager (there's another one for the Style Manager). This is the first thing I tried to use, but it will only open the panel, **and not update the button states**.

If I understand correctly, it's not really like a ``button.click()``, but more like enabling the command bound to the button. What should we do?